src/services/filter/filter_match.rs

```diff
@@ -234,6 +234,9 @@ pub async fn handle_match<T: TriggerExecutionServiceTrait>(
 				)
 				.await;
 		}
+		MonitorMatch::Midnight(_) => {
```

Comment:
This is part of 3rd PR i guess

Reply:
or it is part of src/services/filter/filters/midnight/filter.rs

Reply:
That's right, it's part of #218, most implementations will be either placeholder or empty